website/notifications/utils.py

```diff
@@ -31,6 +32,15 @@ def add_message(self, keys, messages):
         d_to_use['messages'].extend(messages)
 
 
+def find_subscription_type(subscription):
+    """Somewhat robust subscription finder, until something is a subset"""
```

Comment:
This docstring can be improved; describe what the function does and its purpose. Follow PEP 257